arrays/arrays.c

```diff
@@ -72,8 +80,11 @@ void resize_array(Array *arr) {
 char *arr_read(Array *arr, int index) {
 
   // Throw an error if the index is greater than the current count
-
+if(arr->count < index){
+  perror("Array only has %s indices.", arr->count);
```

Comment:
This is good error handling, only thing I'd say is that arr->count is an int, so you should be using %d or %i instead of %s. Also I'm not sure you can use perror() this way. It doesn't seem to work on my system this way, but if it works for you that's fine. Otherwise you could use fprintf(stderr, "Error message")